python/mxnet/gluon/contrib/estimator/event_handler.py

```diff
@@ -263,108 +268,171 @@ def epoch_end(self, estimator, *args, **kwargs):
 
 
 class CheckpointHandler(BatchEnd, EpochEnd):
-    """Save the model after every epoch.
+    """Save the model after user define period
 
-    :py:class:`CheckpointHandler` save the network parameters every epoch
+    :py:class:`CheckpointHandler` saves the network architecture after first batch if the model
+    can be hybridized(), saves model parameters and trainer states after user defined period,
```

Comment:
can be hybridized() -> can be fully hybridized. 
`hybridized()` is not a function